examples/airlift-federation-tutorial/scripts/airflow_setup.sh

```diff
@@ -28,7 +31,106 @@ auth_backend = airflow.api.auth.backend.basic_auth
 [webserver]
 expose_config = True
 web_server_port = $PORT
+secret_key = $SECRET_KEY
+
+EOL
+
+# Create the webserver_config.py file
+cat <<EOL > $AIRFLOW_HOME_DIR/webserver_config.py
```

Comment:
any chance we can curl this sample file and append to it rather than encoding it in a string in the bash file?

Reply:
I think curling it is actually more a risk; since we're using fixed versions of Airflow for testing right now. For a similar reason; I don't think it's a huge deal if we encode it directly.